### ISSUE TITLE
Images not loading in blog Pull Request preview

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#1220]
1) First, I removed the blog post's .md file from https://github.com/GSA/digitalgov.gov/pull/1165 because it was incorrectly dated July 23 and in the /07/ folder. I merged just the eight images (did not delete branch). 

2) Then, I created a new blog post PR here https://github.com/GSA/digitalgov.gov/pull/1219 so that the blog would have correct date and be in correct /08/ folder
 -- but, the images aren't showing in the preview https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/gsa/digitalgov.gov/ToniBonittoGSA-patch-1/2019/08/08/what-pifs-have-been-up-in-2019/ 

Did I miss something, or is that a glitch? The federalist build on the new PR https://github.com/GSA/digitalgov.gov/pull/1219 has been building/pending for almost an hour. 

thanks

**Comment by jeremyzilar** [comment]
OK, I figured out what happened here — 

This PR https://github.com/GSA/digitalgov.gov/pull/1219 was not up to date with master, meaning the images that got pushed live, were not available or accessible to this branch.

Each branch we create is a separate copy of the website. It is possible to update a branch with the code from master, and will figure out how to do it on the website so I can document it.

In short, that is what happened.
It should be working now.

**Comment by ToniBonittoGSA** [comment]
Great, thank you!
closing
